examples/Quasiconvex Optimization by Bisection.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solve the Quasiconvex problem by Convex feasibility and Bisection Search\n",
+    "A quasiconvex problem is defined as - \n",
+    "\n",
+    "\\begin{array}\n",
+    "   \\mbox{minimize}    & f_0(x)\\\\\n",
+    "   \\mbox{subject to}  & f_i(x) \\leq 0  \\\\ \n",
+    "   & Ax = b\n",
+    "\\end{array}\n",
+    "\n",
+    "where $f_0 : \\mathbb{R}^n \\rightarrow \\mathbb{R}$ is Quasiconvex and the remaining functions are convex. \n",
+    "\n",
+    "Here we can have locally optimal solutions that are not globally optimal.\n",
+    " \n",
+    "Quasiconvex functions have the useful property that their sub-level sets are convex. We can exploit this to construct an algorithm to solve a Quasiconvex Problem.\n",
+    "\n",
+    "If $f_0$ is Quasiconvex, then there exists a family of functions $\\phi_t(x)$ that are convex in $x$ for a fixed $t$ such that - $f_0(x) \\leq t \\Leftrightarrow \\phi_t(x) \\leq 0$\n",
+    "\n",
+    "Using this we constuct a convex feasibility problem for a fixed t- \n",
+    "\\begin{array}\n",
+    "\\mbox{Find } & x \\\\ \n",
+    "\\mbox{subject to} &  \\phi_t(x) \\leq 0 \\\\\n",
+    "& f_i(x) \\leq 0  \\\\\n",
+    " & Ax = b\n",
+    "\\end{array}\n",
+    "\n",
```

Comment:
If this problem is feasible, we know that the optimal value of the original quasiconvex optimization problem is no greater than t.